core/chains/evm/log/broadcaster.go

```diff
@@ -614,7 +615,7 @@ func (b *broadcaster) Resume() {
 
 // test only
 func (b *broadcaster) LogsFromBlock(bh common.Hash) int {
-	return len(b.logPool.logsByBlockHash[bh])
+	return b.logPool.TestOnly_getNumLogsForBlock(bh)
 }
```

Comment:
Just move to a `test_helpers.go`?

Reply:
Yes, this is doable. But I didn't want to modify things that are directly not related to the logPool unit-test.
Can do this in a separate PR.